lib/fuel-soap.js

```diff
@@ -555,67 +555,72 @@ FuelSoap.prototype._parseResponse = function(key, body, callback) {
 	};
 
 	parseString(body, parseOptions, function(err, res) {
-		if(err) {
-			err.errorPropagatedFrom = 'xml2js.parseString';
-			callback(err, null);
-			return;
-		}
+		try{
+			if(err) {
+				err.errorPropagatedFrom = 'xml2js.parseString';
+				callback(err, null);
+				return;
+			}
+
+			var soapError;
+			var soapBody = res['soap:Envelope']['soap:Body'];
```

Comment:
I was getting exception from here when `res['soap:Envelope']` is null when res is `<h1>596 Service Not Found</h1>`